content/deploy-guide.md

```diff
@@ -127,7 +129,9 @@ Where `username` is the Galaxy account username the application should deploy in
 
 <h3 id="deployment-token">Using a deployment token</h3>
 
-Galaxy can also accept deployment tokens, which are good for 90 days. You can pass METEOR_SESSION_FILE=token.json before meteor login to generate a login session token so you don’t have to share your login credentials with third-party service providers. This solution is recommended for continuous integration service providers.
+Galaxy can also accept deployment tokens, which are good for 90 days. You can pass `METEOR_SESSION_FILE=token.json` before meteor login to generate a login session token so you don’t have to share your login credentials with third-party service providers. This solution is recommended for continuous integration service providers.
+
+You can use a deployment token as an alternative to typing in your username and password. You'll need to specify it both when running `meteor login` on your machine to generate the file, and when actually running `meteor deploy` in CI. Please note that your organization choice does not affect your deployment token.
```

Comment:
It's more like the other way around — the deployment token doesn't affect the org choice.